test/cypress/integration/03-expenses.page.test.js

```diff
@@ -1,3 +1,155 @@
+
+    // Select Payout Method
+    cy.getByDataCy('payout-method-select').click();
+    cy.contains('[data-cy="select-option"]', 'New custom payout method').click();
+    cy.get('[data-cy="payout-other-info"]').type('A simple thanks would work');
+    // cy.get('input[name="privateMessage"]').type('A simple thanks would work');
+    cy.get('input[name="items[1].description"]').type('Potatoes for the giant raclette');
+    cy.get('input[name="items[1].amount"]').type('{selectall}12.50');
+    cy.getByDataCy('expense-summary-btn').click();
+
+    // Submit!
+    cy.getByDataCy('submit-expense-btn').click();
+    cy.contains('[data-cy="temporary-notification"]', 'Expense submited!');
+    cy.contains('[data-cy="expense-page-content"]', 'May Invoice');
+    cy.getByDataCy('dismiss-temporary-notification-btn').click();
+    cy.getByDataCy('temporary-notification').should('not.exist');
+  });
+
+  it('shows the /expenses page', () => {
+    cy.visit(`/new-collective${collectiveSlug}/expenses/v2`);
+    cy.wait(100);
+    cy.get('[data-cy="single-expense"]').should('have.length', 2);
+    cy.get('[data-cy="single-expense"]:nth-child(1) [data-cy="expense-link"]').click({ force: true });
+    cy.contains('[data-cy="expense-page-content"]', 'May Invoice');
+  });
+
+  it('Filter based on tag', () => {
+    cy.visit(`/new-collective${collectiveSlug}/expenses/v2`);
+    cy.wait(100);
+    cy.get('[data-cy="single-expense"]').should('have.length', 2);
+
+    // Filter based on tag=first
+    cy.get('[data-cy="expense-tags-link"] > div').contains('first').click();
+    cy.get('[data-cy="single-expense"]').should('have.length', 1);
+
+    // Filter based on tag=all
+    cy.get('[data-cy="expense-tags-link"] > div').contains('all').click();
+    cy.get('[data-cy="single-expense"]').should('have.length', 2);
+
+    // Filter based on tag=second
+    cy.get('[data-cy="expense-tags-link"] > div').contains('second').click();
+    cy.get('[data-cy="single-expense"]').should('have.length', 1);
+
+    // Remove all filters
+    cy.get('[data-cy="expense-tags-link"] > div').contains('second').click();
+
+    // Now filter based on reciept
+    cy.get('[data-cy="expense-type-tag"] > div').contains('Receipt').click();
+    cy.get('[data-cy="single-expense"]').should('have.length', 2);
+
+    // Now filter based on inovice
+    cy.get('[data-cy="expense-type-tag"] > div').contains('Invoice').click();
+    // No invoice and hence info message
+    cy.get('[data-cy="zero-expense-message"]').contains('No expense matches the given filters');
+  });
+
+  after(() => {
+    // Delete the collective
+    cy.login({ email: 'testuser+admin@opencollective.com' });
+    cy.visit(`/new-collective${collectiveSlug}/edit/advanced`);
+    cy.contains('button', 'Delete this collective', { timeout: 15000 }).click();
+    cy.get('[data-cy=delete]').click();
+    cy.wait(1000);
+  });
```

Comment:
No need to cleanup

```suggestion
```

Reply:
This is need because `27-host.dashboard.test.js` we are testing pending invoices and other things related to brusselstogetherasbl and hence if I don't clear all this I have to make changes in that test also, Which I think is not need and can be skipped by clearing of this collective. 

Also brusselstogetherasbl can only host 5 collective and in future if more then we need to any way right a logic to delete these and hence I did add them, Let me know If I have to remove this and change logic in `host-dashboard-test.js` also

Reply:
This should not be necessary if using `createHostedCollective` as it's hosting the collective under open source collective, not Brussels Together

Reply:
If I am using `createhostcollective` , then test+useradmin@opencollective is not a admin of the opencollective fiscal host because the collective is created under opencollective fiscal host and hence I cannot approve the expense

To approve I need to be an admin of the fiscal host and test-useradmin is an admin of brussels together

Reply:
I see, let's keep that then. I'll add some more comments to simplify the code